config/olm/kubernetes/0.9.2/manifests/dynatrace-dynakube-oneagent-privileged_v1_serviceaccount.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: v1
+automountServiceAccountToken: false
+kind: ServiceAccount
+metadata:
+  creationTimestamp: null
+  labels:
+    app.kubernetes.io/component: oneagent
+    app.kubernetes.io/name: dynatrace-operator
+    app.kubernetes.io/version: 0.9.0
```

Comment:
is this on `0.9.0` intentionally or should this be `0.9.2` as well? This occurs a few more times in other files as well.